src/apps/robo-vault/common/robo-vault.api.client.ts

```diff
@@ -19,7 +19,7 @@ export class RoboVaultApiClient {
     ttl: 30 * 60,
   })
   async getCachedVaults(network: Network) {
-    const endpoint = 'https://api.robo-vault.com/vaults';
+    const endpoint = ' https://api.v2.robo-vault.com/vaults';
```

Comment:
Is the leading ` ` an error? Does it work like this or should it be removed?

Reply:
my bad it should be removed, did a copy pasta and the linter didn't remove the space😬